### ISSUE TITLE
Please remove or hide this repository

### ISSUE THREAD

**Issue by erikvanoosten** [Paizo/SpringBootCamelStreamsExample#1]
Hello,

I noticed that this repository publishes (parts of) an Ebay copyrighted assignment. Could you please remove or hide this repository?

Kind regards,
    Erik.

**Comment by Paizo** [comment]
Hello Erik,

This is code that I wrote, can you please point me what part of the project is giving you issues?

**Comment by erikvanoosten** [comment]
On further inspection I noticed that there are no direct inclusions of ebay copyrighted material. Sorry for stating that earlier. However, I do see several key data fragments, just slightly changed: files Sheet.csv and Sheet.txt are highly similar to files from the exercise. Furthermore, the rest of the code would be an excellent submission for our exercise. So if you did not write this code in response to our exercise, I am very curious to what happened here.
In short: I can not ask you to remove this, but I am really curious where you got the idea to write this code.

**Comment by Paizo** [comment]
If it is fine for you I would prefer to move the discussion somewhere else, you can reach me at [myGitHubAccountName].gazasoft@gmail.com

**Comment by erikvanoosten** [comment]
Thanks Paizo.
